.circleci/config.yml

```diff
@@ -323,7 +323,7 @@ jobs:
     steps:
       - checkout
       - install-operator-sdk
-      - run: make bundle-validate
+      - run: make bundle
```

Comment:
The original intention was to validate the pushed bundle only, without regenerating it.

If it is going to be regenerated then it would be useful to check that nothing changes.

Reply:
Ok, I will add the check